### PR TITLE
Improve the `pub outdated --help` text

### PR DESCRIPTION
**Pull request by sigurdm**
Fix of https://github.com/dart-lang/pub/issues/2428

I have made the --help text be 80 columns.

@kwalrath The text and arguments has already changed quite a bit since the version you reviewed.
Could you look it through again?

I made a golden-file test of the help text, so we will notice changes.

The golden file (test/outdated/golden/helptext.txt) in this pr can be used for reviewing this.,

I have not changed how the defaults are printed as suggested in the issue.

The (default is on) comes from package:args. And if we want to change them, we should probably change them there. The 'default default' is off, so I guess that is why package:args doesn't print anything in that case.

### CODE REVIEW

**Comment by sigurdm** [comment]
cc @mit-mit

File: lib/src/command/outdated.dart [hunk 1]
```diff
@@ -31,32 +31,40 @@ class OutdatedCommand extends PubCommand {
...
-            'when connected to a terminal, and no-color otherwise.');
-
     argParser.addFlag('json',
-        help: 'Outputs the results in a json formatted report',
+        help: 'Outputs the results in a json formatted\n'
+            'report.',
         negatable: false);
 
+    argParser.addFlag('color',
+        help: 'Whether to color the output.\n'
```
Comment (mit-mit) [inline lib/src/command/outdated.dart:40]: Maybe "Whether to use color in the output.\n"?

File: lib/src/command/outdated.dart [hunk 2]
```diff
@@ -31,32 +31,40 @@ class OutdatedCommand extends PubCommand {
...
         defaultsTo: false,
-        help: 'Include dependencies that are already at the latest version.');
+        help: 'Include dependencies that are already at the\n'
+            'latest version.');
 
-    argParser.addFlag('pre-releases',
+    argParser.addFlag('prereleases',
         defaultsTo: false,
-        help: 'Include pre-releases when reporting latest version.');
+        help: 'Include prereleases when reporting latest\n'
```
Comment (mit-mit) [inline lib/src/command/outdated.dart:51]: Maybe shorten to "Include pre-releases in latest version."

File: lib/src/command/outdated.dart [hunk 3]
```diff
@@ -31,32 +31,40 @@ class OutdatedCommand extends PubCommand {
   String get docUrl => 'https://dart.dev/tools/pub/cmd/pub-outdated';
 
   OutdatedCommand() {
-    argParser.addFlag('color',
-        help: 'Whether to color the output. Defaults to color '
-            'when connected to a terminal, and no-color otherwise.');
-
     argParser.addFlag('json',
-        help: 'Outputs the results in a json formatted report',
+        help: 'Outputs the results in a json formatted\n'
```
[the remark this answers is not in the document]
Comment (sigurdm) [reply lib/src/command/outdated.dart:35]: Done

File: lib/src/command/outdated.dart (hunk 2 above)
Comment (sigurdm) [reply lib/src/command/outdated.dart:51]: Done

File: lib/src/command/outdated.dart (hunk 1 above)
Comment (sigurdm) [reply lib/src/command/outdated.dart:40]: Done

File: lib/src/command/outdated.dart (hunk 3 above)
[the remark this answers is not in the document]
Comment (mit-mit) [reply lib/src/command/outdated.dart:35]: "in a json format" not "in a json formatted"

File: test/outdated/goldens/helptext.txt [hunk 4]
```diff
@@ -0,0 +1,22 @@
+$ pub outdated --help
+Analyze your dependencies to find which ones can be upgraded.
+
+Usage: pub outdated [options]
+-h, --help                         Print this usage information.
+    --json                         Outputs the results in a json formatted
+                                   report.
```
Comment (kwalrath) [inline test/outdated/goldens/helptext.txt:7]:
```suggestion
    --json                         Output the results in JSON format.
```

When the helps starts with a verb, it should be consistently either 3rd person (with an 's' at the end, generally) or 2nd person (no 's'). Most of these commands use 2nd person, so let's do that here.

**Comment by sigurdm** [comment]
I don't think the current order is important
Alphabetical seems fine. Done

File: test/outdated/goldens/helptext.txt (hunk 4 above)
Comment (sigurdm) [reply test/outdated/goldens/helptext.txt]: Done

**Comment by natebosch** [comment]
The args package supports wrapping. Why are we trying to manually line wrap these?

**Pull request merged**